ibis/tests/strategies.py

```diff
@@ -178,22 +178,28 @@ def struct_dtypes(
     return dt.Struct(fields, nullable=draw(nullable))
 
 
-def geometry_dtypes(nullable=_nullable):
-    return st.builds(dt.GeoSpatial, geotype=st.just("geometry"), nullable=nullable)
+def geometry_dtypes(nullable=_nullable, srid=None):
+    return st.builds(
+        dt.GeoSpatial,
+        geotype=st.just("geometry"),
+        nullable=nullable,
+        srid=st.just(srid),
+    )
 
 
 def geography_dtypes(nullable=_nullable):
     return st.builds(dt.GeoSpatial, geotype=st.just("geography"), nullable=nullable)
 
 
 def specific_geometry_dtypes(nullable=_nullable):
+    geotype = st.one_of(st.just("geography"), st.just("geometry"))
     return st.one_of(
-        st.builds(dt.Point, nullable=nullable),
-        st.builds(dt.LineString, nullable=nullable),
-        st.builds(dt.Polygon, nullable=nullable),
-        st.builds(dt.MultiPoint, nullable=nullable),
-        st.builds(dt.MultiLineString, nullable=nullable),
-        st.builds(dt.MultiPolygon, nullable=nullable),
+        st.builds(dt.Point, geotype=geotype, nullable=nullable),
```

Comment:
do we need to add a cases with `srid` not `None` here? or the test with the general Geometry and geography dtype is enough?

Reply:
Just added some, nice catch!

Reply:
as I posted, you fixed this, nice!